### PR TITLE
4 view all articles

### PR DESCRIPTION
**Pull request by ITedInnovator**

### CODE REVIEW

File: src/api.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import axios from "axios";
+
+const instance = axios.create({
+    baseURL: "https://news-reviews-app.onrender.com/api"
+})
+
+export const getAllArticles = () => {
+    return instance.get("/articles");
```
Comment (Slyose) [inline src/api.js]: Usually we'd put the .then directly here to save us doing it elsewhere in the app. Having it here will save us space later on and help separate out the complexity

File: package-lock.json
Comment (Slyose) [inline package-lock.json:1]: should be .gitignored

File: src/App.css [hunk 2]
```diff
@@ -5,38 +5,34 @@
...
+.header-area {
+  flex-wrap: nowrap;
+  width: 60%;
+  margin: 0 auto;
+  justify-content: space-between;
 }
+
+/* a#skip {
+  visibility: hidden;
+} */
```
Comment (Slyose) [inline src/App.css]: Since this is commented out, it shouldn't be included in the final PR.

Try keep it clean!

**Review by Slyose — COMMENTED** [review]
Great for the most part. A few smaller comments to address on here, happy for you to merge solo after reviewing and responding to them.

File: src/api.js (hunk 1 above)
Comment (ITedInnovator) [reply src/api.js]: Thank you for explaining that. Sorted

**Pull request merged**